packages/terra-html-table/src/Table.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Table.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The children passed to the component
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Whether or not the rows should be zebra striped
+   */
+  isStriped: PropTypes.bool,
+  /**
+   * Indicates the desired divider styles to apply to a row and its children.
+   * One of `'none'`, `'vertical'`, `'horizontal'`, `'both'`.
+   */
+  paddingStyle: PropTypes.oneOf(['none', 'standard', 'compact']),
+};
+
+const defaultProps = {
+  isStriped: true,
```

Comment:
With boolean props, we want to default them to false values so consumers can add 
``` <Table isStriped />  ```
 instead of 
 ```<Table isStriped={false} >```

If we want striped to be true, should we flip the prop name to `disabledStripes`?